src/driver.ml

```diff
@@ -1371,6 +1316,31 @@ let standalone () =
     Caml.exit 1
 ;;
 
+let run_as_ppx_rewriter () =
```

Comment:
I'm wondering if we couldn't factorise `standalone_run_as_ppx_rewriter` and `run_as_ppx_rewriter` a bit more. They are starting to look pretty similar. WDYT?

Reply:
I've adapted `run_as_ppx_rewriter` to the workflow of `standalone_run_as_ppx_rewriter` now. That makes it a a little bit harder to read (more lines and less straight-forward), but it makes those two functions even more similar and I don't have to convert between arrays and lists anymore.

Reply:
Update: I've done it the other way around now.